pootle/apps/pootle_store/urls.py

```diff
@@ -29,9 +29,13 @@
         views.BrowseDataDispatcherView.as_view(),
         name='pootle-xhr-stats'),
 
-    url(r'^xhr/units/?$',
-        views.get_units,
-        name='pootle-xhr-units'),
+    url(r'^xhr/uids/?$',
+        views.get_uids,
+        name='pootle-xhr-uids'),
+
+    url(r'^xhr/view-rows/?$',
```

Comment:
These are _units_, which is exactly the data that is being retrieved and it's not ambiguous (I don't envision the uids endpoint getting a `/xhr/units/` name).